src/App.vue

```diff
@@ -157,6 +178,7 @@
         phoneNumber: '0665656565',
         phoneNumber2: null,
         defaultCountry: 'FR',
+        defaultCountryName: '',
```

Comment:
Set the default to the name for FR

Reply:
Sure, missed in the last test case testing